### PR TITLE
N29: Blackjack

### PR DESCRIPTION
**Pull request by setzer22**
_Part of #888_

### CODE REVIEW

File: content/news/029/index.md [hunk 1]
```diff
@@ -73,6 +73,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+- A node-based editor to compose operations like 3d math, vertex/edge/face
+  selections and mesh edit operations.
+- Several polygon edit operations like bevel, chamfer and extrude.
+- Viewport display with support for displaying primitive ids and triangle
+  half-edge winding.
+
+_Discussions:
+[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/rufvlc/meet_blackjack_a_3d_procedural_modelling/),
+[/r/rust](https://www.reddit.com/r/rust/comments/rug24x/media_meet_blackjack_a_3d_procedural_modelling/)_
```
Comment (AngelOnFira) [inline content/news/029/index.md:96]: Move links to the bottom please :)

File: content/news/029/index.md [hunk 2]
```diff
@@ -73,6 +73,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Blackjack]: http://example.com
```
Comment (AngelOnFira) [inline content/news/029/index.md:98]: Template link?

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Comment by setzer22** [comment]
Thanks for the review! :smile: 

Does it look good now?

File: content/news/029/index.md [hunk 3]
```diff
@@ -73,6 +73,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+  selections and mesh edit operations.
+- Several polygon edit operations like bevel, chamfer and extrude.
+- Viewport display with support for displaying primitive ids and triangle
+  half-edge winding.
+
+_Discussions:
+[/r/rust_gamedev](https://reddit.com/r/rust_gamedev/comments/rufvlc/meet_blackjack),
+[/r/rust](https://reddit.com/r/rust/comments/rug24x/media_meet_blackjack)_
+
+[Blackjack](https://github.com/setzer22/blackjack)
```
Comment (ozkriff) [inline content/news/029/index.md:98]: The repository seems empty, is it supposed to be this way?

File: content/news/029/index.md [hunk 4]
```diff
@@ -73,6 +73,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Tooling Updates
 
+### [Blackjack]
+
+![Animated gif showcasing the main features of Blackjack](blackjack.gif)
```
Comment (ozkriff) [inline content/news/029/index.md:78]: Could you please write a more descriptive alt text? Something like "Demo: connecting visual nodes and tweaking various parameters to alter the model in real-time" etc

File: content/news/029/index.md (hunk 3 above)
Comment (setzer22) [reply content/news/029/index.md:98]: For now, yes :+1:  I'm working on an initial release, but I'll make sure to at least have a "coming soon" README by the time this is published.

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**